### ISSUE TITLE
USA - Global Conflicts GUI for Balintawak Blitz timer blocks top choice

### ISSUE THREAD

**Issue by Rambler11555** [TNODevs/TNO-Bug-Tracker#2494]
### Operating System

Windows 10 Home

### HOI4 version

1.12.14

### TNO version

1.5.0a

### Do you have other mods in use?

Radio TNO

### Were you using Steam?

Yes

### Which major expansions do you NOT have?

I have all

### Files

![Blitzbug2](https://github.com/TNODevs/TNO-Bug-Tracker/assets/148181479/69f47f24-5afa-405d-9a48-acf185a46db9)
![BlitzBug1](https://github.com/TNODevs/TNO-Bug-Tracker/assets/148181479/e0318ebb-6517-43a2-99a4-6f76e9b35423)

### Describe the GFX issue

The timer at the top of the conflicts UI ("The Filipino Defense Forces Morale Lowers" for the Balintawak Blitz blocks the top choice - you'll see in the two screenshots, that even though I'm scrolling down the timer remains in place and blocks the view of available decisions. The top decision is completely inaccessible, regardless of how you scroll

### Steps to reproduce / Cause of the issue

Play as America and open the Balintawak Blitz section of the Global Conflicts UI in late March of 1963 

### Expected behaviour

I expected to be able to select the top choice. Usually the timers are integrated with the other decisions, not superimposed over them

### Screenshots

![BlitzBug1](https://github.com/TNODevs/TNO-Bug-Tracker/assets/148181479/ad44eb38-6444-43fe-93c6-b88cec71303b)
![Blitzbug2](https://github.com/TNODevs/TNO-Bug-Tracker/assets/148181479/1cc8dd71-8603-4e39-901f-87071ecd0de3)

**Comment by Rambler11555** [comment]
Actually it's like this for every conflict with a timer now

**Comment by runelead00** [comment]
Working as Designed, the top decision is so other decisions aren't covered

**Comment by Rambler11555** [comment]
It is NOT working as designed! Before the timer started I was able to use the decision that let me trade supplies for command points in Puerto Plata, but now that decision is BLOCKED OUT. It is INACCESSIBLE - this is what you designed? For decisions to be completely unavailable whenever a timed event starts? It is like this for all the events! It used to be that the timers were at the bottom, but now they are at the top and I don't have as many options as I used to have, because certain decisions are BLOCKED
